requirements/edx/github.txt

```diff
@@ -79,7 +79,7 @@ git+https://github.com/edx/XBlock.git@xblock-0.4.12#egg=XBlock==0.4.12
 git+https://github.com/edx/edx-ora2.git@1.1.9#egg=ora2==1.1.9
 -e git+https://github.com/edx/edx-submissions.git@1.1.1#egg=edx-submissions==1.1.1
 git+https://github.com/edx/ease.git@release-2015-07-14#egg=ease==0.1.3
-git+https://github.com/edx/i18n-tools.git@v0.3.2#egg=i18n-tools==v0.3.2
+git+https://github.com/edx/i18n-tools.git@v0.3.4#egg=i18n-tools==v0.3.4
```

Comment:
The i18n errors should have been resolved with #13857. I suggest you rebase, remove this particular line, and see if that helps.